src/hotspot/cpu/aarch64/frame_aarch64.cpp

```diff
@@ -507,8 +507,11 @@ bool frame::is_interpreted_frame_valid(JavaThread* thread) const {
   // do some validation of frame elements
 
   // first the method
-
-  Method* m = *interpreter_frame_method_addr();
+  Method** m_addr = interpreter_frame_method_addr();
+  if (m_addr == nullptr || !os::is_readable_pointer(m_addr)) {
```

Comment:
Just wondering, why checking `is_readable_pointer` + dereference instead of `SafeFetch` which does both in one shot? Especially since `os::is_readable_pointer` is implemented with `SafeFetch` anyway.

Reply:
Thanks for the question. I've just removed the `is_readable_pointer` check since it's not necessary. We only observed a null pointer so the null check is sufficient.

Reply:
Please use SafefetchN, I observed a 0xb value a while back and checking nullptr might not always be sufficient.

Reply:
I implemented @theRealAph's suggestion instead, please take a look